pkg/cloudprovider/providers/vsphere/nodemanager.go

```diff
@@ -39,6 +39,11 @@ type NodeInfo struct {
 	zone       *cloudprovider.Zone
 }
 
+func (n *NodeInfo) String() string {
```

Comment:
I added this in so that NodeInfo gets printed correctly in the controller logs. Otherwise we see only pointer addresses in the logs which is not very useful:
```
I0313 03:23:55.927698       1 nodemanager.go:319] Updated NodeInfo {0xc000617e68 0xc001c872c0 10.161.113.44 42152c0a-5802-4b6f-7e72-d806b0da233d 0xc000f61520} for node "node5".
```